modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -261,6 +261,7 @@ public ObjectEntry addObjectEntry(
 		objectEntry.setCompanyId(user.getCompanyId());
```

Comment:
This is not ordered alphabetically. When I used formatSource, it altered the order I placed set.CreateDate and put it where it is now. It is less ordered now...